src/main/java/com/google/cloud/spanner/pgadapter/wireprotocol/QueryMessage.java

```diff
@@ -115,13 +115,14 @@ public void handleQuery() throws Exception {
                 this.statement.getStatementResult(),
                 this.connection.getServer().getOptions(),
                 QueryMode.SIMPLE)
-            .send();
+            .send(false);
       }
       this.sendSpannerResult(this.statement, QueryMode.SIMPLE, 0L);
       boolean inTransaction = connection.getSpannerConnection().isInTransaction();
       new ReadyResponse(
               this.outputStream, inTransaction ? Status.TRANSACTION : ReadyResponse.Status.IDLE)
-          .send();
+          .send(false);
+      this.outputStream.flush();
```

Comment:
Is this change necessary? I mean if we would've left line 124 as it was and didn't add line 125, then also it would have been the same.

Reply:
Good point, changed.